src/components/NcRichText/NcRichText.vue

```diff
@@ -321,12 +320,22 @@ import { RouterLink } from 'vue-router'
 /**
  * Heavy libraries should be loaded on demand to reduce component size
  */
+let remarkGfm
+const remarkGfmLoaded = ref(false)
+/**
+ * Load 'remark-gfm' library when prop `useExtendedMarkdown` is truthy
+ */
+async function importRemarkGfmLibrary() {
+	const module = await import('remark-gfm')
+	remarkGfm = module.default
+	remarkGfmLoaded.value = true
+}
```

Comment:
Just realized, we only need one variable here.

```suggestion
const remarkGfm = ref(null)
/**
 * Load 'remark-gfm' library when prop `useExtendedMarkdown` is truthy
 */
async function importRemarkGfmLibrary() {
	const module = await import('remark-gfm')
	remarkGfm.value = module.default
}
```